lib/src/televerse/televerse.dart

```diff
@@ -1213,7 +1213,7 @@ class Televerse<TeleverseSession extends Session> {
     HandlerScope scope = HandlerScope<ChatMemberUpdatedHandler>(
       handler: callback,
       types: [
-        UpdateType.chatMember,
+        UpdateType.myChatMember,
```

Comment:
This one is `UpdateType.chatMember` itself,  https://github.com/HeySreelal/televerse/blob/b26ecd7c8ca18ce18df33ba6d1b9e79cf8de2ce9/lib/src/televerse/televerse.dart#L1199-L1210 should be changed to `UpdateType.myChatMember`